src/mumble/PluginInstaller.cpp

```diff
@@ -194,6 +210,10 @@ QString PluginInstaller::getInstallDir() {
 	return Global::get().qdBasePath.absolutePath() + QLatin1String("/Plugins");
 }
 
+QLatin1String PluginInstaller::getPluginSuffix() {
+	return QLatin1String("_" MUMBLE_TARGET_OS "_" MUMBLE_TARGET_ARCH);
```

Comment:
```suggestion
	return QLatin1String("_" MUMBLE_TARGET_OS "." MUMBLE_TARGET_ARCH);
```

Reply:
Why a dot? The docs already state to use an underscore :thinking:

Reply:
Ah, I see. It was just for consistency with the binaries we release.